components/director/internal/tenantfetchersvc/handler_test.go

```diff
@@ -52,6 +52,17 @@ func TestService_SubscriptionFlows(t *testing.T) {
 	})
 	assert.NoError(t, err)
 
+	validRequestBodyWithMatchingParentID, err := json.Marshal(regionalTenantCreationRequest{
```

Comment:
isn't this the same as `bodyWithMatchingAccountAndSubaccountIDs ` (with missing new prop, ocf)? we can rename the variable to this one, but the test case should be already covered